python/sglang/srt/layers/rotary_embedding.py

```diff
@@ -141,17 +149,25 @@ def forward_cuda(
         key: torch.Tensor,
         offsets: Optional[torch.Tensor] = None,
     ) -> Tuple[torch.Tensor, torch.Tensor]:
-        from vllm import _custom_ops as ops
-
-        self.cos_sin_cache = self.cos_sin_cache.to(query.device, dtype=query.dtype)
-        ops.rotary_embedding(
-            positions,
-            query,
-            key,
-            self.head_size,
-            self.cos_sin_cache,
-            self.is_neox_style,
-        )
+        if _is_cuda_available:
+            apply_rope_with_cos_sin_cache_inplace(
```

Comment:
Everything is going well except for an accuracy issue with [test_session_control](https://github.com/sgl-project/sglang/blob/main/test/srt/test_session_control.py).